mule-extensions-api/src/main/java/org/mule/runtime/extension/api/annotation/dsl/xml/XmlHints.java

```diff
@@ -47,4 +47,9 @@
    */
   boolean allowReferences() default true;
 
+  /**
+   * Added if the element created from these annotations should extend any substitutionGroup
```

Comment:
Reword as: `Specifies that the annotated element should extend a particular substitution group`. Also explain how namespaces are to be handled.